src/chainstate/stacks/db/contracts.rs

```diff
@@ -0,0 +1,127 @@
+/*
+ copyright: (c) 2013-2019 by Blockstack PBC, a public benefit corporation.
+
+ This file is part of Blockstack.
+
+ Blockstack is free software. You may redistribute or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License or
+ (at your option) any later version.
+
+ Blockstack is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY, including without the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Blockstack. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+use std::io;
+use std::io::prelude::*;
+use std::fmt;
+use std::fs;
+use hashbrown::HashMap;
```

Comment:
If we're not using`hashbrown::HashSet`, we can get rid of this `hashbrown` dependency, it became the default implementation of `HashMap` with Rust 1.36 - https://github.com/rust-lang/hashbrown

Reply:
Ah, didn't know that.  Sounds good to me :+1:

Reply:
Fixed as of now.